starter_code/queries.md

```diff
@@ -4,80 +4,109 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+    query: {name: "Babelgum"}
+    projection: {name: 1}
```

Comment:
es, and if you don't want the id field, you have to state it explicitly : Project: {name: 1, _id: 0}